pmd.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    name="Android Application Rules"
+    xmlns="http://pmd.sf.net/ruleset/1.0.0"
+    xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd"
+    xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd" >
+
+    <description>PMD rules</description>
+
+    <rule ref="category/java/errorprone.xml">
+        <exclude name="BeanMembersShouldSerialize"/>
+    </rule>
+    <rule ref="category/java/bestpractices.xml">
+        <exclude name="JUnitTestsShouldIncludeAssert"/>
+        <exclude name="JUnitAssertionsShouldIncludeMessage"/>
+        <exclude name="JUnitTestContainsTooManyAsserts"/>
```

Comment:
Is it inline with programme standards to exclude these?

Reply:
Following upgrades in other projects I've done some further tidying up on here too, put the pmd version back as what it was originally and rather than starting to do work on that tech debt in this ticket, simply set the max permitted failures to the current level. I understand there's already a tech debt ticket for updating all the projects to use the new code analysis tools so hesitant to do extra unrelated work on this ticket that may soon become redundant anyway.

Reply:
Could we please link that tech debt ticket to this one as a related one?